agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -95,10 +96,26 @@ function collect_sos {
 	fi
 }
 
-collect_kernel_config &
-collect_libvirt &
-collect_topology &
-collect_sos &
-collect_block &
+if [[ -n "$sysinfo" ]]; then
+	for item in ${sysinfo//,/ };do
+        	if [[ "$item" == "default" ]] || [[ "$item" == "all" ]]; then
+			collect_kernel_config &
```

Comment:
So I think we want to allow for `default` and `all` to be specified alone, but not along side other sysinfo options.  Then before we get to this point, we convert `default` and `all` to the appropriate lists that we iterate over.

Make sense?